kernelci/build.py

```diff
@@ -802,6 +805,22 @@ def _kernel_config_enabled(self, config_name):
         cmd = 'grep -cq CONFIG_{}=y {}'.format(config_name, dot_config)
         return shell_cmd(cmd, True)
 
+    def _kernel_config_getkey(self, opt_name):
+        dot_config = os.path.join(self._output_path, '.config')
+        with open(dot_config, 'r') as fp:
+            for line in fp:
+                if line.startswith(opt_name):
+                    pcfg = line.strip().split('=')
+                    if len(pcfg) > 1 and pcfg[0] == opt_name:
+                        return pcfg[1]
+        return False
```

Comment:
I think it could make more sense to return `None` here.
You either return a value if the option exists or `None` if it doesn't it'll also resemble more how `.get()` method works for dicts and it's always good to keep the API consistent.

> There should be one-- and preferably only one --obvious way to do it.  [Zen of Python]

Reply:
Updated according suggestions